README.md

```diff
@@ -90,12 +90,12 @@ All benchmarks run on a local dev machine are highly suspect.  If you want
 more scientific these should really be done in a production server environment.
 
 ### Benchmarking Phoenix
-Elixir 0.14.2
+Elixir 1.0.3
 
 ```bash
 $ mix do deps.get, compile
 $ MIX_ENV=prod mix compile.protocols
-$ MIX_ENV=prod elixir -pa _build/prod/consolidated -S mix phoenix.start
+$ MIX_ENV=prod elixir -pa _build/prod/consolidated -S mix phoenix.server
 Running Elixir.Benchmarker.Router with Cowboy on port 4000
```

Comment:
I don't get this line of output anymore (process starts silently).  Am I doing something wrong?